tests/parm/model_configure_regional_stoch.IN

```diff
@@ -0,0 +1,38 @@
+start_year:              2019
+start_month:             5
+start_day:               30
+start_hour:              0
+start_minute:            0
+start_second:            0
+nhours_fcst:             @[FHMAX]
+fhrot:                   @[FHROT]
+
+dt_atmos:                36
+calendar:                'julian'
+restart_interval:        @[RESTART_INTERVAL]
+output_1st_tstep_rst:    .false.
+
+quilting:                @[QUILTING]
+write_groups:            1
+write_tasks_per_group:   12
+itasks:                  @[ITASKS]
+write_dopost:            @[WRITE_DOPOST]
+num_files:               2
+filename_base:           'dyn' 'phy'
+output_file:             @[OUTPUT_FILE]
+ideflate:                @[IDEFLATE]
+
+output_grid:             'lambert_conformal'
+cen_lon:  -97.5
+cen_lat:  35.0
+stdlat1:  35.0
+stdlat2:  35.0
+nx:       837
+ny:       595
+lon1:     -109.97410429
+lat1:     26.31459843
```

Comment:
I'm just curious, is this some standard output grid, how did you come up with the lon1,lat1 values with so many significant digits? Would lon1=-110.0, lat1=26.0 or 26.5 work?

Reply:
Good question.  The reason the write component is so precise is related to how it's generated.  @gsketefian used NCL scripts to create write component grids that fit symmetrically within the native grid.  In order to make that work, the precision is necessarily pretty high.  In this case, the write component grid is one grid point from the native grid on all sides.

Reply:
But this is Lambert Conformal grid, while the native grid is ESG, how can Lambert grid be exactly one grid point from the native grid on all sides, when they are not even on the same projection? At most one or two corner points can be exactly one grid point from the corner or the edge of the ESG grid, the edges are not parallel to each other. But I do not think that's very important, to be exactly one dx or dy away from the edge of the native grid.

Reply:
@DusanJovic-NOAA, I should have said, "In this case, the write component grid is one grid point from the native grid at the point where they are closest", but @gsketefian can elaborate.

Reply:
@DusanJovic-NOAA The NCL scripts that generate these numbers take in as input a parameter named `num_margin_cells` that specifies the minimum number cells to include between the boundaries of the two grids (the scripts might actually add 1 to this number for safety; I'd have to go back and look carefully).  But the significant digits do need to be kept in order for the write-component grid to be placed such that the margins along the south, east, north, and west boundaries are all at least `num_margin_cells` cells wide and that the write-component grid is placed symmetrically in the east-west direction (to within the same number of digits in longitude and latitude, in this case 8 digits) with respect to the native grid.  I've just cut and pasted these numbers from the output of the NCL scripts; ideally, this would all be done internally and the accuracy automatically maintained, but that's not automated yet.  Below are figures showing an example of a grid over Peru:  (a) the whole grid, (b) the northeast corner, and (c) the southeast corner.  Red is the boundary of the native grid, blue is the boundary of the write-component grid, and the orange cells are the cells in the halo.
![C3346_grids SAPeru_whole](https://user-images.githubusercontent.com/31046882/154375681-25feae42-b9cd-43ef-acbd-2d9d8b821595.png)
![C3346_grids SAPeru_NEcorner](https://user-images.githubusercontent.com/31046882/154375691-7084d925-3dfd-4584-99e6-5d0d6d7a7d82.png)
![C3346_grids SAPeru_SEcorner](https://user-images.githubusercontent.com/31046882/154375716-3bc92f92-bc46-44b1-835b-5523071791f8.png)

Reply:
Can that routine round the values to let's say the nearest thousand of a degree? For example would lon1=-110.0 or lon1=-109.97 not fit? Does ten-thousand of a degree matter that much. I'm asking because the above values are read in as double precision, but written out as single precision floats. Furthermore if these fields need to be saved in grib (by post for example) you wont get full precision either, the values will be rounded by encoding program.